pkg/meta/meta.go

```diff
@@ -1489,6 +1524,98 @@ func (m *Meta) SetSchemaDiff(diff *model.SchemaDiff) error {
 	return errors.Trace(err)
 }
 
+// TableNameKey constructs the key for table name.
+func (*Meta) TableNameKey(dbName string, tableName string) kv.Key {
+	var sb strings.Builder
+	sb.Write(mNames)
+	sb.WriteByte(':')
+	sb.WriteString(strings.ToLower(dbName))
+	sb.Write(mNameSep)
+	sb.WriteString(strings.ToLower(tableName))
+	return kv.Key(sb.String())
+}
+
+// CheckTableNameExists checks if the table name exists.
+func (m *Meta) CheckTableNameExists(name []byte) error {
+	v, err := m.txn.Get(name)
+	if err == nil && v == nil {
+		err = ErrTableNotExists.GenWithStack(string(name))
```

Comment:
```suggestion
		err = ErrTableNotExists.GenWithStackByArgs(string(name))
```

It is different. `GenWithStack` will overwrite the error message.

Reply:
BTW `FastGen` is enough. We don't need the stack for client errors.